File: java/delivery-api/GetType.java
// DocSection: delivery_api_get_type
// Tip: Find more about Java SDK at https://kontent.ai/learn/java
import kontent.ai.delivery.*;

// Initializes a DeliveryClient
DeliveryClient client = new DeliveryClient("<YOUR_PROJECT_ID>");

// Gets a content type
CompletionStage<ContentType> type = client.getType("article");
// EndDocSection

File: php/delivery-api/GetType.php
// DocSection: delivery_api_get_type
<?php
// Tip: Find more about PHP SDKs at https://docs.kontent.ai/php

// Defined by Composer to include required libraries
require __DIR__ . '/vendor/autoload.php';

use Kentico\Kontent\Delivery\DeliveryClient;

$client = new DeliveryClient('<YOUR_PROJECT_ID>');

$type = $client->getType('coffee');
// EndDocSection